packages/framework/lib/modules/index.ts

```diff
@@ -90,6 +90,20 @@ export function addTextRule<U extends User<any>>(
     }
 }
 
+export function addReferalsRule<U extends User<any>>(
+    module: Module<U>,
+    action: (user: U, payload: IPayload, ...args: any[]) => Promise<any>,
+    rule: RegExp
+): void {
+    if (module.actions === undefined) {
```

Comment:
Παραλείψαμε κάποια πράγματα?? || !(actionName in module.actions))
Για να τσεκάρει ότι το action έχει προστεθεί πριν πας να ορισεις τον κανόνα

Reply:
Με actionName const actionName = module.name + '/' + action.name;